tkp/steps/forced_fitting.py

```diff
@@ -83,25 +83,27 @@ def perform_forced_fits(fit_posns, fit_ids,
         NB returned lists may be shorter than input lists
         if some fits are unsuccessful.
     """
-    logger.info("Forced fitting in image: %s" % (image_path))
-    fitsimage = tkp.accessors.open(image_path)
-
-    data_image = sourcefinder_image_from_accessor(fitsimage,
-                    margin=extraction_params['margin'],
-                    radius=extraction_params['extraction_radius_pix'],
-                    back_size_x=extraction_params['back_size_x'],
-                    back_size_y=extraction_params['back_size_y'])
-
-
-    boxsize = extraction_params['box_in_beampix'] * max(data_image.beam[0],
-                                             data_image.beam[1])
-    successful_fits, successful_ids = data_image.fit_fixed_positions(
-                                                fit_posns, boxsize, ids=fit_ids)
+    logger.info("Forced fitting in image: %s" % (accessor.url))
+
+    margin = extraction_params['margin']
+    radius = extraction_params['extraction_radius_pix']
+    back_size_x = extraction_params['back_size_x']
+    back_size_y = extraction_params['back_size_y']
+    data_image = sourcefinder_image_from_accessor(accessor, margin=margin,
```

Comment:
I noticed, my PyCharm is one big Christmas party when i open your code ;) But to be fair, also most of my own old code, jeez I hate my old coding me sometimes. I think in general it gives cleaner code and leaves less space for ambiguity if people just follow the pep8. For new projects I do a pep8 check next to the test run and let travis fail if a PR is not pep8 compatible.